openfl-workspace/torch_cnn_mnist/README.md

```diff
@@ -121,6 +120,43 @@ class MNISTDataset(ImageFolder):
         else:
             return super(MNISTDataset, self).__getitem__(index)
 ```
+Use a pre-sharded version of the standard MNIST dataset (divided among two data owners). The pre-sharded dataset can be downloaded from [mnist_data_shards.tar.gz](https://github.com/securefederatedai/openfl-contrib/blob/main/openfl_contrib_tutorials/ml_to_fl/federated/mnist_data_shards.tar.gz). 
+
+Copy the dataset bundle to the root of the FL workspace and unpack it:
+```
+cp mnist_data_shards.tar.gz ~/openfl/openfl-workspace/torch_cnn_mnist/
+cd ~/openfl/openfl-workspace/torch_cnn_mnist/
+tar -xvf mnist_data_shards.tar.gz
+rm mnist_data_shards.tar.gz
+```
+This will populate the data folder of the FL workspace with two shards (data/1 and data/2) of labeled MNIST images of digits (the 0–9 labels being encoded in the sub-folder names). 
```

Comment:
data folder -> `/data` folder

Reply:
fixed in c9bb19088383b978a147804539dee0d7dd37b515